src/index.scss

```diff
@@ -0,0 +1,235 @@
+@mixin center {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  height: 100%;
+}
+
+body {
+  margin: 0;
+  color: #ffeead;
+  background: #ffeead;
+}
+
+.containter {
+  display: grid;
+  grid-template-areas:
+    'logo     header'
+    'sidebar  content'
+    'sidebar  content'
+    'footer   footer';
+  grid-template-rows: 50px minmax(calc(50vh - 75px), 100%) minmax(calc(50vh - 75px), 100%) 50px;
+  grid-template-columns: 150px 1fr;
+  grid-gap: 5px;
+  min-height: calc(100vh - 30px);
+  margin: 15px;
+  font-family: monospace;
+  font-size: 22px;
+  font-weight: 600;
+
+  &__logo {
+    grid-area: logo;
+    background: #96ceb4;
+
+    &-title {
+      @include center;
+
+      text-transform: uppercase;
```

Comment:
As we see, you have just 1 place where text is Capitalized (Menu nav). In all other places the text is UPPERCASE. It makes sense to set the text-transform: uppercase; globally, to body for example. And set it Capitalized to menu nav only. In this case you will save 3 lines of code.

Reply:
ok